pdr_backend/lake/test/test_partitioned_data_store.py

```diff
@@ -0,0 +1,65 @@
+import os
+import polars as pl
+from pdr_backend.lake.partitioned_data_store import (
+    PartitionedDataStore,
+)  # Adjust the import based on your project structure
+
+
+# Initialize the PartitionedDataStore instance for testing
+def _get_test_manager(tmpdir):
+    return PartitionedDataStore(str(tmpdir))
+
+
+def test_append_data(tmpdir):
+    test_manager = _get_test_manager(tmpdir)
+    example_df = pl.DataFrame(
+        {"timestamp": ["2022-01-01", "2022-02-01", "2022-03-01"], "value": [10, 20, 30]}
+    )
+    dataset_identifier = "test_append"
+    test_manager.append_data(example_df, dataset_identifier)
+
+    # Check if the view is registered
+    view_name = test_manager._generate_view_name(str(tmpdir) + dataset_identifier)
+    tables = test_manager.duckdb_conn.execute(
+        "SELECT table_name FROM information_schema.tables WHERE table_schema = 'main'"
+    ).fetchall()
+    assert view_name in [table[0] for table in tables]
+    # Since append_data doesn't immediately reflect in the filesystem, this test is limited
```

Comment:
Please show that we can continue to append data, and that the records in the db are increasing.
We should be able to call "select * from dataset_identifier" and see the `len(records)` growing.
As-is, right now, I don't believe this is working.